### DISCUSSION TITLE
Server header image not changing?

### DISCUSSION THREAD

**Discussion by grubcode** [GameServerManagers/LinuxGSM#3447] [Q&A]
I've tried to do it in linuxgsm. I have tried 1024 x 512 and 512 x 256 image sizes. I did put jpg at the end of my imgur link, it still does change. Any solution to this?

**Comment by wldwalker** [comment]
Need a lot more information. What server are you running? Sounds like this might be more specific to the game server and not LGSM.

**Reply by grubcode** [reply]
I'm running a Rust server.
